src/commands/load-all.ts

```diff
@@ -30,6 +30,14 @@ function getRegister<F extends Register.Flags | Register.Flags[]>(
   return (argument.register = register as any);
 }
 
+function getInputLength(argument: { inputLength?: number }) {
+  const len = argument.inputLength || 1;
```

Comment:
If `argument.inputLength` is `0`, this will yield `1`. I think using the same logic as `getCount` (using `+` to cast to a number and then checking that `len > 0 && Number.isInteger(len)` would be better (so if the input is a string for some reason, that still works).